src/Widgets/DataPersonCardWidget/DataPersonCardWidgetComponent.tsx

```diff
@@ -1,26 +1,39 @@
-import { connect, DataItem, provideComponent, useData } from 'scrivito'
+import { connect, ContentTag, DataItem, provideComponent } from 'scrivito'
 
 import { DataBinaryImage } from '../../Components/DataBinaryImage'
 import { ensureString } from '../../utils/ensureString'
 import { isDataBinary } from '../../utils/dataBinaryToUrl'
 import { DataPersonCardWidget } from './DataPersonCardWidgetClass'
 import { EditorNote } from '../../Components/EditorNote'
 import personCircle from '../../assets/images/person-circle.svg'
+import { CurrentUser } from '../../Data/CurrentUser/CurrentUserDataItem'
+import { User } from '../../Data/User/UserDataClass'
 import { Loading } from '../../Components/Loading'
 
 provideComponent(
   DataPersonCardWidget,
-  () => {
-    const dataScope = useData()
+  ({ widget }) => {
+    // TODO: Remove workaround, once #10883 is resolved
+    const name = ensureString(widget.get('attributeName'))
+    if (!name) return <EditorNote>Data is empty.</EditorNote>
 
-    if (dataScope.isEmpty()) return <EditorNote>Data is empty.</EditorNote>
+    const value = CurrentUser.get(name)
```

Comment:
* [ ] FYI instead of hardcoding this to CurrentUser and an `attributeName: 'string'` we could use `data: 'datalocator'`, let the user pick the data attribute, and access it by `value = useDataLocator(widget.get('data')).dataItemAttribute()?.get()`.
<img width="216" alt="image" src="https://github.com/Scrivito/scrivito-portal-app/assets/986170/6e1504f4-cf9c-4ca8-9fac-612f25f4a25c">

Reply:
Thanks for the pointer. `const dataScope = useDataLocator(widget.get('data'))` also works.